examples/tf-s3-consumer/pom.xml

```diff
@@ -4,15 +4,15 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <properties>
-    <tf-maven-version>0.8.2</tf-maven-version>
+    <tf-maven-version>0.9.1</tf-maven-version>
```

Comment:
Change the version to 0.10-SNAPSHOT, pls and update the link in the README to show that the latest SNAPSHOT release is 0.10-SNAPSHOT. When it's merged into the develop branch, it will auto deploy the SNAPSHOT to SonaType's SNAPSHOT repo.